app/controllers/onboarding_controller.rb

```diff
@@ -3,11 +3,12 @@
 class OnboardingController < ApplicationController
   skip_before_action :authenticate
   before_action :verify_token
+  skip_before_action :verify_token, only: %i[create_invite_url]
```

Comment:
```suggestion
  before_action :verify_token, except: :create_invite_url
```

Reply:
oh, yeah, I was pretty sure there was an `except`, but didn't find it in the docs...